docs/emcc.txt

```diff
@@ -141,9 +141,12 @@ Options that are modified or new in *emcc* are listed below:
 "-gseparate-dwarf[=FILENAME]"
    Preserve debug information, but in a separate file on the side.
    This is the same as "-g", but the main file will contain no debug
-   info, while debug info will be present in a file on the side
-   ("FILENAME" if provided, otherwise the same as the wasm file but
-   with suffix ".debug.wasm").
+   info, while debug info will be present in a file on the side,
+   "FILENAME" if provided, otherwise the same as the wasm file but
+   with suffix ".debug.wasm". You can use "-s
+   SEPARATE_DWARF_URL=PATH_IN_FILE" to customize the path to the debug
```

Comment:
It's not clear why this is called "PATH_IN_FILE".
Because it's the path that goes in the wasm file?
The doc should maybe be a little more clear in that when you use gseparate-dwarf, the main wasm includes whatever path you put in FILENAME (correct? even if it's an absolute path?), and then it becomes more clear exactly what SEPARATE_DWARF_URL means.

Reply:
I tried to clarify that better now, WDYT?